torch/csrc/jit/mobile/flatbuffer_loader.h

```diff
@@ -15,6 +15,7 @@
 namespace torch {
 namespace jit {
 
+using ExtraFilesMap = std::unordered_map<std::string, std::string>;
```

Comment:
if there an ExtraFilesMap definition somewhere else? If so reuse that one.

Reply:
It is not available while importing in mobile
 So it is redefined in [mobile/import.h](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/mobile/import.h#L14)
  
Once we integrate flatbuffer we can reuse the definition from mobile/import

Reply:
Sorry why cannot this file #include <torch/csrc/jit/mobile/import.h>? import.h is not part of mobile?